cddl/condition-triple-record.cddl

```diff
@@ -0,0 +1,4 @@
+condition-triple-record = [
+  environment-map
+  measurement-map
+]
```

Comment:
this needs to be explicitly added to the CDDL fragments' list to be visible

Reply:
so I thought.... actually fixed in https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/174/commits/c261262fcadb5c76e62b648a0d030cfeea80d19a